src/library/library.h

```diff
@@ -18,6 +18,17 @@
 #include "analysisfeature.h"
 #include "library/coverartcache.h"
 #include "library/setlogfeature.h"
+#include "library/crate/cratefeature.h"
+#include "library/rhythmbox/rhythmboxfeature.h"
+#include "library/banshee/bansheefeature.h"
+#include "library/recording/recordingfeature.h"
+#include "library/itunes/itunesfeature.h"
+#include "library/mixxxlibraryfeature.h"
+#include "library/autodj/autodjfeature.h"
+#include "library/playlistfeature.h"
+#include "library/setlogfeature.h"
+#include "library/traktor/traktorfeature.h"
+#include "library/browse/browsefeature.h"
```

Comment:
Are these included still required? Can they be moved to the cpp file?

Reply:
Of course, they're essential to initialize the library features.
Though these includes have been in the cpp file earlier, I can move them back.
Just notice there was also a double include: setlogfeature.h

Reply:
It is a good idea to keep the includes as close as possible to the usage, to limit the files that need to rebuild if something changes in the header.

Reply:
oki. all feature includes go to .cpp